#!/usr/bin/env python
PACKAGE = "arducam_camera"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("exposureValue",int_t, 0,"Exposure Value ", 3, 3, 64)

exit(gen.generate(PACKAGE, "arducam_camera", "arducam_camera_dynamic"))